docs/Guides/General guides/Combat.md

```diff
@@ -72,10 +72,10 @@ Fighting groups can take two routes, depending on the form of the group. a rarer
 Well, you killed someone, or a lot of people. Turns out, killing people can have repercussions. This section will be about making sure you dont end up like your victims in the aftermath. There are typically three routes you can go down, with varying degrees of legality and possibility for revenge.
 
 ## Taking responsibility
-You see that fella you killed? time to unkill him. This is the best way to avoid the wrath of \[sec](Security Officer), the guy you killed and any admins, though it isnt a good option for when you \[need them dead](Antagonist) for your employers and may be impossible to do depending on how wanted you are already. The best case scenario is dragging them to cloning yourself, second best is leaving them outside medbay, and third best is just dragging them out into the hall and leaving them there for someone else to clone. 
+You see that fella you killed? time to unkill him. This is the best way to avoid the wrath of [sec](Security Officer.md), the guy you killed and any admins, though it isnt a good option for when you [need them dead](Antagonist.md) for your employers and may be impossible to do depending on how wanted you are already. The best case scenario is dragging them to cloning yourself, second best is leaving them outside medbay, and third best is just dragging them out into the hall and leaving them there for someone else to clone. 
 
 ## The null outcome
-Simply dont touch the body after your done. This can be taken as a sign of mercy, incompetence, or mocking, depending on the whims of the person you killed. It wont let you avoid the wrath of \[sec](Security Officer), and can leave the guy you killed mad, but it can be enough to make admins brush it aside, so long as you had reason to kill them in the first place. Definitely not the best option, but sometimes necessary, especially if your on the run already.
+Simply dont touch the body after your done. This can be taken as a sign of mercy, incompetence, or mocking, depending on the whims of the person you killed. It wont let you avoid the wrath of [sec](Security Officer.md), and can leave the guy you killed mad, but it can be enough to make admins brush it aside, so long as you had reason to kill them in the first place. Definitely not the best option, but sometimes necessary, especially if your on the run already.
```

Comment:
```suggestion
Simply dont touch the body after your done. This can be taken as a sign of mercy, incompetence, or mocking, depending on the whims of the person you killed. It wont let you avoid the wrath of [sec](Security-Officer.md), and can leave the guy you killed mad, but it can be enough to make admins brush it aside, so long as you had reason to kill them in the first place. Definitely not the best option, but sometimes necessary, especially if your on the run already.
```